xapian-letor/api/featurelist_internal.h

```diff
@@ -178,7 +183,7 @@ class FeatureList::Internal : public Xapian::Internal::intrusive_base {
 			    const Xapian::Document & doc);
 
     /// Computes and populates the stats needed by a Feature.
-    void populate_feature(Feature *feature_);
+    void populate_featurelist(Feature::Internal *internal_feature);
```

Comment:
This should probably be called `populate_feature_internal()` or something now. Also, it's only called from `FeatureList`, which is already a friend class, so does it have to be public?

Reply:
I will change that in a fixup commit.